Interni/NormeProgetto/components/norme-c3.tex

```diff
@@ -0,0 +1,475 @@
+	\item la raccolta di norme da rispettare durante la stesura dei documenti.
+\end{itemize}
+\subsubsection{Ciclo di vita di un documento}
+Ogni documento redatto passa per le seguenti fasi:
+\begin{itemize}
+	\item \textbf{Creazione} del documento tramite template;
+	\item \textbf{Strutturazione} del documento tramite registro delle modifiche,  indice di contenuti e/o indice di figure e tabelle;
+	\item \textbf{Stesura} del corpo del documento,  scritto da più membri del gruppo;
+	\item \textbf{Revisione} del documento, gestita da un \textit{Verificatore} che,  regolarmente,  riesamina le sezioni del suddetto;
+	\item \textbf{Approvazione} del documento, stabilita dal \textit{Responsabile di Progetto} quando la revisione risulta positiva; consente il rilascio del documento.
+\end{itemize}
+\subsubsection{Documenti prodotti \hfil}
+
+I documenti prodotti saranno di due tipi:
+\begin{itemize}
+	\item \textbf{Formali}: sono caratterizzati da storico delle versioni prodotte e da approvazione della versione definitiva da parte del \textit{Responsabile di Progetto}. Possono essere:
+	\begin{itemize}
+		\item \textbf{interni}: riguardanti le dinamiche del gruppo;
+		\item \textbf{esterni}: di interesse ai committenti e al proponente, vengono consegnati nell'ultima versione approvata.
+	\end{itemize} 
+In particolare, i documenti formali prodotti saranno:
+\newpage
+{
+	\setlength{\freewidth}{\dimexpr\textwidth-1\tabcolsep}
+	\renewcommand{\arraystretch}{1.5}
+	\setlength{\aboverulesep}{0pt}
+	\setlength{\belowrulesep}{0pt}
+	\rowcolors{2}{AzzurroGruppo!10}{white}
+	\begin{longtable}{L{.210\freewidth} L{.6\freewidth} L{.080\freewidth}}
+		\toprule 
+		\rowcolor{AzzurroGruppo!30}
+		\textbf{Nome} & \textbf{Descrizione} & \textbf{Uso}\\
+		\toprule
+		\endhead		
+		\NdP{} & Contiene tutte le regole stabilite dai membri alle quali attenersi durante l'intera durata del progetto. & Interno \\ 
+		\PdP{} & Contiene la pianificazione di tutte le attività previste, comprendente il preventivo delle spese e una previsione dell'impegno in ore per ogni membro del gruppo. & Esterno \\
+		\PdQ{} & Descrive i criteri di valutazione della qualità impiegate dal gruppo & Esterno \\ 
+		\SdF{} & Contiene l'analisi dei capitolati messi a disposizione, evidenziandone pregi e difetti. Contiene, inoltre, il capitolato scelto dal gruppo & Interno \\
+		\AdR{} & Descrive i requisiti che il prodotto dovrà possedere per essere in linea con le richieste dei committenti. & Esterno \\ 	
+		\G{} & Elenco di tutti i termini presenti nella documentazione che, secondo i membri, necessitano una definizione al fine di chiarirne il significato e rimuovere eventuali ambivalenze. & Esterno \\  			
+		\bottomrule
+		\hiderowcolors
+		\caption{Nome, Descrizione ed uso dei documenti formali prodotti}
+	\end{longtable}
+}
+
+\item \textbf{Informali}: sono documenti non soggetti a versionamento o che non sono ancora stati approvati dal \textit{Responsabile di Progetto}. \newline 
```

Comment:
cambiare 
".. \textit{Responsabile di Progetto} ..."
in
"... \RdP{} ..."

Reply:
Fixed